Platform/TigerlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -356,6 +391,34 @@ UpdateFspConfig (
   Fspmcfg->BiosGuard = 0x0;               // Need disable, else it will failed in fSPS
   Fspmcfg->SafeMode = 0x1;                // Need enable, else failed in MRC
 
+  // TXT Configuration
+  if (FeaturePcdGet (PcdTxtEnabled)) {
+    FeaturesCfgData = (FEATURES_CFG_DATA *) FindConfigDataByTag(CDATA_FEATURES_TAG);
+    if (FeaturesCfgData->Features.TxtEnabled == 1) {
+      DEBUG((DEBUG_INFO, "Enabling TXT in FSP-M UPD's\n"));
+      Fspmcfg->Txt                  = 0x1;
+      Fspmcfg->TxtImplemented       = 0x1;
+      Fspmcfg->SinitMemorySize      = 0x50000;
+      Fspmcfg->TxtHeapMemorySize    = 0xF0000;
+      Fspmcfg->BiosAcmBase          = (UINT32)(UINTN)FindBiosAcm();
+      FspmUpd->FspmConfig.VmxEnable = 1;    // Txt need enable VMX
```

Comment:
No need to use "FspmUpd", please simply use:  

  Fspmcfg->VmxEnable = 1;

Reply:
done